Description: upstream: autotools: help2man
 Incorporate within  the autotool chain the generation of
 a manpage for the iprange front-end program with help2man(1);
 meant to be submitted to the upstream maintainer.
Origin: debian
Forwarded: https://github.com/firehol/iprange/pull/12
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2015-12-27

--- a/configure.ac
+++ b/configure.ac
@@ -32,6 +32,7 @@
 AC_CANONICAL_HOST
 AC_PROG_CC
 AC_PROG_INSTALL
+AC_PATH_PROG([HELP2MAN],[help2man])
 AC_ARG_ENABLE(
 	[pedantic],
 	[AS_HELP_STRING([--enable-pedantic], [enable pedantic compiler warnings])],
--- a/Makefile.am
+++ b/Makefile.am
@@ -14,10 +14,17 @@
 	$(srcdir)/m4/ltsugar.m4 $(srcdir)/m4/libtool.m4 \
 	$(srcdir)/m4/ltversion.m4 $(srcdir)/m4/lt~obsolete.m4 \
 	$(srcdir)/m4/ltoptions.m4 \
-	$(srcdir)/pkcs11-helper.spec $(srcdir)/config-w32-vc.h
+	$(srcdir)/pkcs11-helper.spec $(srcdir)/config-w32-vc.h \
+	iprange.1
+
+AM_H2MFLAGS = \
+	--no-info
+
+iprange_DESCRIPTION = "manage IP ranges"
 
 bin_PROGRAMS = iprange
 dist_noinst_DATA = iprange.spec
+man_MANS = iprange.1
 
 iprange_SOURCES = iprange.c
 
@@ -25,5 +32,15 @@
 	.gitignore \
 	README.md \
 	iprange-9999.ebuild \
-        autogen.sh \
+  autogen.sh \
+	iprange.1 \
 	$(NULL)
+
+%.1: %
+	$(HELP2MAN) \
+			-s 1 \
+			$(AM_H2MFLAGS) \
+			-I $(top_srcdir)/$*.h2m \
+			-n $(if $($(subst -,_,$*)_DESCRIPTION), $($(subst -,_,$*)_DESCRIPTION), "manual page for $*") \
+			-o $@ \
+		$(top_builddir)/$<
